docs/manual/core-extensions/calendar/frontend-modules.en.md

```diff
@@ -101,14 +97,14 @@ The following calendar templates are available by default:
 
 ## Event reader
 
-The frontend module "Eventleser" is used to display a specific event. The module obtains the alias of the entry via the URL, so that events can be linked with so-called [permalinks](https://de.wikipedia.org/wiki/Permalink):
+The frontend module "Event reader" is used to display a specific event. The module obtains the alias of the entry via the URL, so that events can be linked with [permalinks](https://wikipedia.org/wiki/Permalink):
```

Comment:
```suggestion
The front end module "Event reader" is used to display a specific event. The module obtains the alias of the entry via the URL, so that events can be linked with [permalinks](https://wikipedia.org/wiki/Permalink):
```

Reply:
Agreed